3p/nameframe.max.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Style nit: [Needs to be lowercase.](https://google.github.io/styleguide/htmlcssguide.xml?showone=Capitalization#Capitalization) Same with UTF-8 below.